### ISSUE TITLE
DMA2D_Config

### ISSUE THREAD

**Issue by epikao** [lvgl/lv_port_stm32h7b3i_disco#2]
Hello,
In the code there is a DMA2D_Config()... however this is not used, but why? 
Then what is it intended for?

What setup is the official video based on, see link below?
DMA, DMA_IT, DMA2D ? Or something else? Pixel format 32bit or 16bit ?

https://blog.lvgl.io/2022-12-08/stm32h7b3i-review

Thank you

**Comment by davidanderle** [comment]
Yes, I have a feeling some files are missing or are not up-to-date

**Comment by davidanderle** [comment]
https://github.com/lvgl/lv_port_stm32h7b3i_disco/blob/95da448d8eae6f25a8e0e9ddc109036c8b0011e4/hal_stm_lvgl/tft/tft.c
In this commit the DMA2D_Config still exists. I guess along the way the DMA functionality was lost
